node/resolvers/directives/helper.ts

```diff
@@ -2,7 +2,8 @@ import { isUserPartOfBuyerOrg } from '../Queries/Users'
 
 export const validateAdminToken = async (
   context: Context,
-  adminUserAuthToken: string
+  adminUserAuthToken: string,
+  orgPermission?: 'buyer_organization_edit' | 'buyer_organization_view'
```

Comment:
(lets change `orgPermission` everywhere to `requiredPermission`
```suggestion
  requiredPermission?: 'buyer_organization_edit' | 'buyer_organization_view'
```